Lab3/priority_queue.cpp

```diff
@@ -31,22 +32,28 @@ bool PriorityQueue::full() const {
 
 // PURPOSE: Prints the contents of the priority queue; format not specified
 void PriorityQueue::print() const {
+    cerr<<"order: Root->Left->Right"<<endl;
```

Comment:
this isn't totally true, if you're printing it in the order of the indexes the it's printing all the nodes of each depth one after the other (so like root->left->right would print the whole left tree before the whole right tree, which isn't what we're doing)

Reply:
also fyi test #2 isn't done that was more just me trying to debug lol

Reply:
okay, that makes sense, is that what it is doing? so its not inOrder, postOder or preOrder?